pxd_bio_blkmq.c

```diff
@@ -273,7 +285,9 @@ static int prep_root_bio(struct fp_root_context *fproot) {
         bio->bi_size = 0;
 #endif
         bio->bi_end_io = stub_endio; // should never get called
+#if defined(bio_copy_dev)
```

Comment:
Please use the same macro here based on version and encapsulate both init on bio as below within.
```
#if LINUX_VERSION_CODE < KERNEL_VERSION(5,18,0) /// to sync up with the usage of newer bio_alloc_bioset.
  BIO_COPY_DEV(bio, rq->bio);
  BIO_SET_OP_ATTRS(bio, BIO_OP(rq->bio), op_flags);
#endif
```